taxi/urls.py

```diff
@@ -37,15 +40,62 @@
         ManufacturerDeleteView.as_view(),
         name="manufacturer-delete",
     ),
-    path("cars/", CarListView.as_view(), name="car-list"),
-    path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
-    path("cars/create/", CarCreateView.as_view(), name="car-create"),
-    path("cars/<int:pk>/update/", CarUpdateView.as_view(), name="car-update"),
-    path("cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"),
-    path("drivers/", DriverListView.as_view(), name="driver-list"),
     path(
-        "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
+        "cars/",
+        CarListView.as_view(),
+        name="car-list"
+    ),
+    path(
+        "cars/<int:pk>/",
+        CarDetailView.as_view(),
+        name="car-detail"
+    ),
+    path(
+        "cars/create/",
+        CarCreateView.as_view(),
+        name="car-create"
+    ),
+    path(
+        "cars/<int:pk>/update/",
+        CarUpdateView.as_view(),
+        name="car-update"
+    ),
+    path(
+        "cars/<int:pk>/delete/",
+        CarDeleteView.as_view(),
+        name="car-delete"
+    ),
+    path(
+        "drivers/",
+        DriverListView.as_view(),
+        name="driver-list"
+    ),
+    path(
+        "drivers/<int:pk>/",
+        DriverDetailView.as_view(),
+        name="driver-detail"
+    ),
+    path(
+        "drivers/create/",
+        DriverCreateView.as_view(),
+        name="driver-create"
+    ),
+    path(
+        "drivers/<int:pk>/delete/",
+        DriverDeleteView.as_view(),
+        name="driver-delete"
+    ),
+    path(
+        "drivers/<int:pk>/update/",
+        DriverUpdateView.as_view(),
+        name="driver-update"
+    ),
+    path(
+        "cars/<int:pk>/assign-car/",
+        assign_car,
+        name="assign-car"
```

Comment:
not sure about this name, since it tells me that we assign car to somebody, while function makes something different.